contracts/payloads/IGP30/PayloadIGP30.sol

```diff
@@ -0,0 +1,876 @@
+    function action4() internal {
+        // wstETH/wBTC
+        {
+            VaultConfig memory vaultConfig = VaultConfig({
+                // user supply config for the vault on Liquidity Layer.
+                supplyToken: wstETH_ADDRESS,
+                supplyMode: 1, // Mode 1
+                supplyExpandPercent: 25 * 1e2, // 25%
+                supplyExpandDuration: 12 hours, // 12 hours
+                supplyBaseLimit: 1875 * 1e18, // ~$7507575 @ $4,004.04/wstETH
+
+                borrowToken: wBTC_ADDRESS,
+                borrowMode: 1, // Mode 1
+                borrowExpandPercent: 20 * 1e2, // 20%
+                borrowExpandDuration: 12 hours, // 12 hours
+                borrowBaseLimit: 122 * 1e8, // ~$7552129 @ $61,902.70/wBTC
+                borrowMaxLimit: 324 * 1e8, // ~$20056474 @ $61,902.70/wBTC
+
+                supplyRateMagnifier: 100 * 1e2, // 1x
+                borrowRateMagnifier: 100 * 1e2, // 1x
+                collateralFactor: 88 * 1e2, // 88% 
+                liquidationThreshold: 91 * 1e2, // 91% 
+                liquidationMaxLimit: 94 * 1e2, // 94% 
+                withdrawGap: 5 * 1e2, // 5% 
+                liquidationPenalty: 2 * 1e2, // 2% 
+                borrowFee: 0 * 1e2, // 0% 
+
+                oracle: address(0)
+            });
+
+            // Deploy wstETH/wBTC
+            deployVault(vaultConfig);
+        }
+
+        // weETH/wBTC
+        {
+            VaultConfig memory vaultConfig = VaultConfig({
+                // user supply config for the vault on Liquidity Layer.
+                supplyToken: weETH_ADDRESS,
+                supplyMode: 1, // Mode 1
+                supplyExpandPercent: 25 * 1e2, // 25%
+                supplyExpandDuration: 12 hours, // 12 hours
+                supplyBaseLimit: 2107 * 1e18, // // ~$7501615 @ $3,560.33/wstETH 
+
+                borrowToken: wBTC_ADDRESS,
+                borrowMode: 1, // Mode 1
+                borrowExpandPercent: 20 * 1e2, // 20%
+                borrowExpandDuration: 12 hours, // 12 hours
+                borrowBaseLimit: 122 * 1e8, // ~$7552129 @ $61,902.70/wBTC
+                borrowMaxLimit: 324 * 1e8, // ~$20056474 @ $61,902.70/wBTC
+
+                supplyRateMagnifier: 100 * 1e2, // 1x
+                borrowRateMagnifier: 100 * 1e2, // 1x
+                collateralFactor: 80 * 1e2, // 80% 
+                liquidationThreshold: 85 * 1e2, // 85% 
+                liquidationMaxLimit: 90 * 1e2, // 90% 
+                withdrawGap: 5 * 1e2, // 5% 
+                liquidationPenalty: 5 * 1e2, // 5% 
+                borrowFee: 0 * 1e2, // 0% 
+
+                oracle: address(0)
+            });
+
+            // Deploy weETH/wBTC
+            deployVault(vaultConfig);
+        }
+    }
+
+    /// @notice Action 5: Clone from old vault config to new vault
+    function action5() internal {
+        for (uint oldVaultId = 1; oldVaultId <= 10; oldVaultId++) {
```

Comment:
I think we better clone configs from current vaults now and just set the new borrow & withdraw limits?

Reply:
ignore this, current way is correct, cloning live limits of old vaults